sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -1853,7 +1853,7 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
     }
   }
 
-  test("SPARK-18352: Expect one JSON document per file") {
+  ignore("SPARK-18352: Expect one JSON document per file") {
```

Comment:
Why this test is ignored? I know the behaviour is changed but  what about adapting this test/ deleting if it is not needed (or adding a comment with the reason of the ignore).